rust/main/utils/run-locally/src/server.rs

```diff
@@ -0,0 +1,61 @@
+use std::io;
+
+use reqwest::Url;
+use serde::{Deserialize, Serialize};
+
+use crate::RELAYER_METRICS_PORT;
+
+/// Copied from agents/relayer/src/server/message_retry.rs
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
+pub struct MessageRetryResponse {
+    /// ID of the retry request
+    pub uuid: String,
+    /// how many pending operations were evaluated
+    pub evaluated: usize,
+    /// how many of the pending operations matched the retry request pattern
+    pub matched: u64,
+}
+
+pub fn run_retry_request() -> io::Result<MessageRetryResponse> {
```

Comment:
can you add some comments here / call_retry_request so to just say what the intended behavior / use is here (e.g. mentioning that it retries all messages)